src/Tribe/Main.php

```diff
@@ -80,7 +80,7 @@ class Tribe__Events__Main {
 		public $postOrganizerTypeArgs = array();
 
 		public static $tribeUrl = 'http://tri.be/';
-		public static $tecUrl = 'http://theeventscalendar.com/';
+		public static $tecUrl = 'https://theeventscalendar.com/';
```

Comment:
This is used as a base url to concatenate into various places, so m.tri.be wouldn't work. We could review those locations and switch it out, but that's more of a lift than I think is worth it at the moment.